src/__tests__/Raids.test.tsx

```diff
@@ -21,8 +21,7 @@ const data = fetchedDocumentData.data
 
 it('should test the ViewRaid $TotalStats', async () => {
   render(<App />)
-
-  userEvent.click(screen.getAllByRole('link', { name: /raids/i })[1])
+  userEvent.click(screen.getByTestId('raids'))
```

Comment:
I'm noting this down, so that it becomes a conscious decision.

Ideally we would be using `*ByRole` and the name parameter, but with how much this is likely to change, this will be far better for moving faster.